src/lib/components/workspace/member-worspace/MemberWorkspace.svelte

```diff
@@ -0,0 +1,145 @@
+<script lang="ts">
+  import MemberDropdown from "$lib/components/dropdown/MemberDropdown.svelte";
+  import { TeamRole, WorkspaceRole } from "$lib/utils/enums/team.enum";
+  import type { TeamServiceMethods } from "$lib/utils/interfaces";
+  import { notifications } from "$lib/utils/notifications";
+  export let workspace;
+  export let user;
+  export let userType;
+  export let teamServiceMethods: TeamServiceMethods;
+  export let userId: string;
+
+  const handleDropdown = async (id) => {
+    if (id === "remove") {
+      // perform remove perations
```

Comment:
correct the spelling in the comment to avoid confusions

Reply:
Spell corrected